includes/class-frontend-form-post.php

```diff
@@ -650,16 +666,19 @@ public static function update_post_meta( $meta_vars, $post_id ) {
         // prepare the meta vars
         list( $meta_key_value, $multi_repeated, $files ) = self::prepare_meta_fields( $meta_vars );
         // set featured image if there's any
-        $wpuf_files = isset( $_POST['wpuf_files'] ) ? wp_unslash( $_POST['wpuf_files'] ) : [];
+
+        // @codingStandardsIgnoreStart
+        $wpuf_files = isset( $_POST['wpuf_files'] ) ? sanitize_text_field( wp_unslash( $_POST['wpuf_files'] ) ) : [];
```

Comment:
This should be an array, sanitizing with  text field shouldn't be done I think. It may change to array, please check